FAQ.md

```diff
@@ -9,34 +9,50 @@ category: FAQ
   + [关于产品](#关于产品)
     - [TiDB 是什么？](#tidb-是什么)
     - [TiDB 是基于 MySQL 开发的吗？](#tidb-是基于-mysql-开发的吗)
+    - [TiDB 和 MySQL Group Replication 的区别是什么？](#TiDB和-MySQL-Group-Replication的区别是什么)
     - [TiDB 和 TiKV 是如何配合使用？ 他们之间的关系是？](#tidb-和-tikv-是如何配合使用-他们之间的关系是)
     - [Placement Driver (PD) 是做什么的？](#placement-driver-pd-是做什么的)
     - [TiDB 用起来简单吗？](#tidb-用起来简单吗)
     - [TiDB 适用的场景是？](#tidb-适用的场景是)
     - [TiDB 不适用于哪些场景？](#tidb-不适用于哪些场景)
     - [TiDB 的强一致特性是什么样的？](#tidb-的强一致特性是什么样的)
     - [TiDB 支持分布式事务吗？](#tidb-支持分布式事务吗)
+    - [当多个事务冲突严重时（如同时修改同一行数据），会造成某些事务写入失败吗？](#当多个事务冲突严重时-如同时修改同一行数据-会造成某些事务写入失败吗)
     - [在使用 TiDB 时，我需要用什么编程语言？](#在使用-tidb-时我需要用什么编程语言)
     - [和 MySQL/Oracle 等传统关系型数据库相比，TiDB 有什么优势？](#和-mysqloracle-等传统关系型数据库相比tidb-有什么优势)
     - [和 Cassandra/Hbase/MongoDB 等 NoSQL 数据库相比，TiDB 有什么优势？](#和-cassandrahbasemongodb-等-nosql-数据库相比tidb-有什么优势)
     - [使用 `go get` 方式安装 TiDB 为什么报错了？](#使用-go-get-方式安装-tidb-为什么报错了)
     - [TiDB 高可用的特性是怎么样的？](#tidb-高可用的特性是怎么样的)
+    - [TiDB 中删除数据后会立即释放空间吗？](#TiDB-中删除数据后会立即释放空间吗)
+    - [Load 数据时可以对目标表执行 DDL 操作吗？](#Load数据时可以对目标表执行DDL操作吗)
+    - [TiDB 是否支持 replace into 语法？](#TiDB是否支持replace-into语法)
+    - [如何导出 TiDB 数据？](#如何导出-unload-TiDB数据)
+    - [TiDB 是否支持会话超时？](#TiDB-是否支持会话超时)
   + [PD](#pd)
     - [访问 PD 报错：`TiKV cluster is not bootstrapped`](#访问-pd-报错tikv-cluster-is-not-bootstrapped)
     - [PD 启动报错：`etcd cluster ID mismatch`](#pd-启动报错etcd-cluster-id-mismatch)
     - [更改 PD 的启动参数](#更改-pd-的启动参数)
     - [PD 能容忍的时间同步误差是多少？](#pd-能容忍的时间同步误差是多少)
+    - [Client 连接是如何与寻找 PD 的？](#Client-连接是如何与寻找PD的)
+    - [PD 参数中 leader-schedule-limit 和 region-schedule-limit 调度有什么区别？](#PD参数中leader-schedule-limit和region-schedule-limit调度有什么区别)
+    - [每个 region group 的 replica 数量可配置吗？调整的方法是？](#每个region-group的replica数量可配置吗调整的方法是)
```

Comment:
I think you mean "Raft Group" here. If so, please update.

Reply:
should be "region", corrected.